azkaban-common/src/main/java/azkaban/utils/AbstractMailer.java

```diff
@@ -38,6 +39,7 @@
   public AbstractMailer(Props props) {
     this.azkabanName = props.getString("azkaban.name", "azkaban");
     this.mailHost = props.getString("mail.host", "localhost");
+    this.mailPort = props.getInt("mail.port", Emailer.DEFAULT_SMTP_PORT);
```

Comment:
It seems a bit weird for AbstractMailer class to depend on one of its subclasses (Emailer).  How about defining the constant DEFAULT_SMTP_PORT inside AbstractMailer class?